daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -286,6 +286,8 @@ class Engine(val config: EngineConfig = Engine.StableConfig) {
       sexpr <- runCompilerSafely(
         NameOf.qualifiedNameOfCurrentFunc,
         compiledPackages.compiler.unsafeCompile(commands),
+        // TODO: enable priming on ledger cached contract map with disclosed contracts
```

Comment:
FIXME: this is probably the wrong place to do this - look at Speedy.from* functions instead and use compiledPackages.compiler from compilation?